src/components/molecules/MoleculeFeatureCard.vue

```diff
@@ -37,7 +37,7 @@ export default {
 
 .custom-card:hover,
 .custom-card:focus {
-    background-color: var(--primary-color);
+    background-color: #ffe08a;
```

Comment:
don't hardcode here... change the --primary-color to #ffe08a in assets/styles/variables.css